docs/website/scripts/update-plugins-docs/file-system.ts

```diff
@@ -0,0 +1,13 @@
+import { readFileSync, readdirSync, writeFileSync } from 'fs'
+
+const UTF8: BufferEncoding = 'utf-8'
+
+export const readJSON = (filePath: string): void =>
```

Comment:
":void" is wrong
Can you configure .pre-commit & github CI to validate this scripts?

Reply:
I have set it up! It's working correctly 👍